### PR TITLE
BC-3347 - refactor title element

### PR DESCRIPTION
**Pull request by christian-darsow**
# Short Description

To be consistent the title card element should be a property similar to dueDate and visibleAtDate - and not a card element anymore

## Links to Ticket and related Pull-Requests
https://ticketsystem.dbildungscloud.de/browse/BC-3347

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/pages/TaskCard.page.vue [hunk 1]
```diff
@@ -233,42 +207,36 @@ export default defineComponent({
 			});
 		};
 
+		// TODO improve with regular frontend validation, needed for now to satisfy backend validation
+		const validate = (content: string) => {
+			return content.length > 2;
```
Comment (MartinSchuhmacher) [inline src/pages/TaskCard.page.vue:212]: this one will change with what we discussed yesterday in BC-2807, I guess you should talk to @odalys-dataport if you already remove it or we leave it for now to meet current BE state

**Review by MartinSchuhmacher — APPROVED** [review]

File: src/pages/TaskCard.page.vue (hunk 1 above)
Comment (christian-darsow) [reply src/pages/TaskCard.page.vue:212]: Yeap, that should change. Lets leave it for this PR like this and change in the upcoming tickets

File: src/components/card-elements/CardElementWrapper.unit.ts [hunk 2]
```diff
@@ -177,15 +106,6 @@ describe("@components/card-elements/CardElementWrapper", () => {
...
 
-		it("should render title component", () => {
-			const wrapper = getWrapper({
-				component: CardElementComponentEnum.Title,
-				editMode: false,
-			});
-
-			expect(wrapper.findComponent(TitleCardElement).exists()).toBe(true);
-		});
-
```
Comment (odalys-dataport) [inline src/components/card-elements/CardElementWrapper.unit.ts:188]: Why were all these tests removed?

Comment (christian-darsow) [reply src/components/card-elements/CardElementWrapper.unit.ts:188]: the title is not handled by the ContentWrapper any longer

**Pull request merged**